repository/repository-server/src/test/java/org/eclipse/vorto/repository/web/core/PayloadMappingControllerTest.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.vorto.repository.web.core;
+
+import org.eclipse.vorto.repository.server.it.AbstractIntegrationTest;
+import org.eclipse.vorto.repository.server.it.TestModel;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class PayloadMappingControllerTest extends AbstractIntegrationTest {
```

Comment:
This test class seems not be done as it contains empty test methods. Am I missing sth ?

Reply:
this is a work in progress branch, to track what we are currently each working on

Reply:
this is still a in progress thing should contain test tomorrow night